examples/redis/v1beta3/image/redis-master.conf

```diff
@@ -0,0 +1,827 @@
+# Redis configuration file example
```

Comment:
If these .conf files are identical to the v1beta1 versions, I'm inclined to not copy them. Just reuse the existing ones.